subclassScript.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
What's this file for?

Reply:
Was testing to confirm where the error was in the script, see issue https://github.com/UCSolarCarTeam/Epsilon-Raspberry/issues/10

Reply:
This should get removed then